src/views/Jobs/JobResultsTable.vue

```diff
@@ -841,6 +879,14 @@ export default Vue.extend({
                     uuid: card.uuid,
                     geneId: geneId
                   });
+                  this.$store.commit("interactiveMap/updateKnockoutReaction", {
+                    uuid: card.uuid,
+                    gene: {
+                      id: geneId,
+                      name: geneId,
```

Comment:
Guess we should rather leave name empty (e.g. "N/A") than pretend the ID is the same as the name